x-pack/plugins/lists/common/schemas/types/endpoint/entry_nested.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
What's this for?

Reply:
I think i grabbed an extra line when copy pasting the header...good catch

Will remove in all the places